code/datums/dog_fashion.dm

```diff
@@ -10,6 +10,8 @@
 	// sprite that the associated item uses
 	var/icon_file
 	var/icon_state
+	var/alpha
+	var/color
```

Comment:
imo change these var names to prevent confusion with inbuilt atom alpha and color vars.